src/translations/screens/subscreens/PurchaseConfirmation.ts

```diff
@@ -46,6 +46,10 @@ const PurchaseConfirmationTexts = {
         `This ticket will be valid from ${time}.`,
       ),
   },
+  travelDate: {
```

Comment:
I see that this is moved from another place. Did we miss the a11y label in the transition?

Reply:
I think the a11yHint was just a remnant after we redid the ticketing flow a while back, which is why it was placed in PurchaseOverview instead of PurchaseConfirmation. After the change it's not clickable anymore, which is why it doesn't need a a11yHint.